package.json

```diff
@@ -19,6 +19,7 @@
     "formik": "^2.2.6",
     "i18next": "^20.2.1",
     "i18next-browser-languagedetector": "^6.1.0",
+    "lodash.isequal": "^4.5.0",
```

Comment:
veldig mye å sette seg inn i, så klarer ikke helt å få kontroll over PRen, men aldri et godt tegn at man tar inn lodash https://medium.com/techspiration/why-you-shouldnt-use-lodash-f8504d7b7383

Reply:
Den er kun tatt inn for å ha en generisk isEquals(a ,b) funksjon. Vil du anbefale å lage en egen isEquals metode for å sammenligne queryObject eller en annen pakke? (jeg fant bare lodash som gjorde jobben).

Reply:
ville kanskje først se om det er mulig å skrive seg helt vekk fra behovet. Er det feks nødvendig å samle alle settings i et eget query object? kan man feks løse det samme bare ved å beholde verdiene utenfor noe felles objekt? 
det vil (muligens) forenkle noen useEffects også, hvis du vil trigge siden man da sier eksplisitt hvilke verdier vi er opptatt av, i stedet for å trigge ny useEffect når hva som helst i objektet endrer seg

Reply:
Skrevet meg bort fra lodash behovet og flere av useEffectene er fjernet